syft/pkg/cataloger/binary/test-fixtures/config.yaml

```diff
@@ -830,3 +830,14 @@ from-images:
         platform: linux/arm64
     paths:
       - /usr/local/bin/ffmpeg
+
+  - name: ffmpeg-shared-libs
```

Comment:
This config is used to get snippets, but there are no snippets I see, rather the Dockerfile. we could/should probably stick to the same pattern with snippets if possible.